### PR TITLE
Add webViewWebContentProcessDidTerminate delegate method to WKWebView…

### PR DESCRIPTION
**Pull request by olofd**
…-bridge.

This adds missing delegate method to wkwebview-bridge. The method is called when wkwebview crashes.

### CODE REVIEW

File: WebViewJavascriptBridge/WKWebViewJavascriptBridge.m [hunk 1]
```diff
@@ -158,6 +158,13 @@ - (void)webView:(WKWebView *)webView didFailProvisionalNavigation:(WKNavigation
     }
 }
 
+- (void)webViewWebContentProcessDidTerminate:(WKWebView *)webView {
```
Comment (marcuswestin) [inline WebViewJavascriptBridge/WKWebViewJavascriptBridge.m]: Should this also have `if (webView != _webView) { return; }`?

**Comment by marcuswestin** [comment]
Also, looks like the test is failing: https://travis-ci.org/marcuswestin/WebViewJavascriptBridge/builds/133902448

**Comment by marcuswestin** [comment]
Closing due to failed test:
```No known instance method for selector 'webViewWebContentProcessDidTerminate:'```
Feel free to reopen with additional changes.

**Comment by liuwin7** [comment]
Please delegate the method `- (void)webViewWebContentProcessDidTerminate:(WKWebView *)webView` out to the `_webViewDelegate`.
When the content of a WKWebView crash, the `_webView` is still exists. I need to deal with this situation by `[webview reload]`.
As for the WKWebView Content crash, you can refer [Blank screen after running app for awhile](https://github.com/Telerik-Verified-Plugins/WKWebView/issues/41).
Many thanks.

File: WebViewJavascriptBridge/WKWebViewJavascriptBridge.m (hunk 1 above)
Comment (liuwin7) [reply WebViewJavascriptBridge/WKWebViewJavascriptBridge.m]: When WKWebView content crashed, the `_webView` is no been released yet!

**Pull request closed without merging**